locale/en/admin.po

```diff
@@ -318,6 +318,19 @@ msgstr "Introduction"
 msgid "admin.settings.minPasswordLength"
 msgstr "Minimum password length (characters)"
 
+msgid "admin.settings.sharedReviewerStatistics"
+msgstr "Reviewer statistics"
+
+msgid "admin.settings.sharedReviewerStatistics.description"
+msgstr "On a multi-journals OJS instance, the reviewer statistics displayed "
+"in the review request window (number of reviews submitted, one active or "
+"not, etc.) can be either for all journals hosted on the OJS instance or "
+"specific to the journal only. By default, reviewer statistics counts are for "
+"all journals hosted."
```

Comment:
The `pkp-lib` is shared with other applications (OJS/OMP/OPS), so we should avoid using specific terms (e.g. OJS/journal), otherwise we'll need to customize these texts for each application.

Perhaps `In a multi-context installation, reviewer statistics, such as the count of submitted reviews, can be displayed either individually for each context or aggregated collectively`